### PR TITLE
docker: remove github.com/docker/engine-api

### PR DESCRIPTION
**Pull request by runcom**
Close https://github.com/projectatomic/skopeo/issues/239

Signed-off-by: Antonio Murdaca <runcom@redhat.com>

### CODE REVIEW

File: vendor.conf [hunk 1]
```diff
@@ -11,14 +11,13 @@ github.com/pmezard/go-difflib master
...
+github.com/docker/docker v1.13.0
 github.com/docker/go-connections 4ccf312bf1d35e5dbda654e57a9be4c3f3cd0366
-github.com/vbatts/tar-split v0.9.11
+github.com/vbatts/tar-split v0.10.1
 github.com/gorilla/context 14f550f51a
 github.com/gorilla/mux e444e69cbd
-github.com/docker/go-units 651fc226e7441360384da338d0fd37f2440ffbe3
-golang.org/x/net master https://github.com/golang/net.git
+github.com/docker/go-units 8a7beacffa3009a9ac66bad506b18ffdd110cf97
+golang.org/x/net 2beffdc2e92c8a3027590f898fe88f69af48a3f8 https://github.com/tonistiigi/net.git
```
Comment (mtrmac) [inline vendor.conf]: Is this really necessary? That fork is based on a version of `x/net` from April 2016, which is pretty old for such a critical infrastructure package.  Besides, the change only affects `x/net/trace`/, which is stripped by `vndr` as unused anyway.

Comment (runcom) [inline vendor.conf]: W/o this tests were failing because of this package. I'll try without, maybe I overlooked something

Comment (mtrmac) [inline vendor.conf]: … I am being stupid; of course that the fork only _changes_ `x/net/trace` does not mean that the two have no diverged in the intervening 9 months.

Comment (mtrmac) [inline vendor.conf]: FWIW a local build changing this to `golang.org/x/net master` does succeed for me.

Comment (runcom) [inline vendor.conf]: fixed

**Comment by mtrmac** [comment]
ACK, thanks!

**Pull request merged**